HISTORY.rst

```diff
@@ -3,6 +3,23 @@
 History
 =======
 
+1.1.3 (2022-11-03)
+------------------
+
+New features:
+
+- Add support for fractional units (#876)
+
+Improvements:
+
+- Update workflow versions (#1088)
+- Fix returned datetime skips a day with time+timezone input and PREFER_DATES_FROM = 'future' (#1002)
+- Fix Ukrainian supplemental data to be in the YAML and restore CLDR data (#1090)
+- Fix translate keep_formatting (#720)
+- Add support for the text "till date" for the language English (#1005)
+- Added support for after and before relative dates (#1008)
```

Comment:
What about moving cleanups to a separate section (less relevant to end users) and a bit of rewording to clarify some of the changes?

```suggestion
- Fix the returned datetime skipping a day with time+timezone input and PREFER_DATES_FROM = 'future' (#1002)
- Fix input translate breaking keep_formatting (#720)
- English: support "till date" (#1005)
- English: support “after” and “before” in relative dates (#1008)

Cleanups:

- Reorganize internal data (#1090)
- CI updates (#1088)
```